pkg/skaffold/deploy/kubectl/kubectl.go

```diff
@@ -145,10 +135,11 @@ func (k *KubectlDeployer) manifestFiles(manifests []string) ([]string, error) {
 
 	if len(gcsManifests) != 0 {
 		// return tmp dir of the downloaded manifests
-		tmpDir, err := downloadManifestsFromGCS(gcsManifests)
+		tmpDir, err := manifest.DownloadManifestsFromGCS(gcsManifests)
```

Comment:
`DownloadManifestsFromGCS` to `DownloadFromGCS`

Reply:
👍